app/views/layouts/application.html.erb

```diff
@@ -9,11 +9,18 @@
 <p class="notice"><%= notice %></p>
 <p class="alert"><%= alert %></p>
 
-<ul class="nav nav-tabs">
-  <li><button class="btn btn-default" type="submit"><%= link_to 'Log In' , new_user_session_path %></button></li>
-  <li><button class="btn btn-default" type="submit"><%= link_to 'Log Out', logout_path, method: :delete %></button></li>
-  <li><button class="btn btn-default" type="submit"><%= link_to 'Sign Up', new_user_registration_path %></button></li>
-  <li><button class="btn btn-default" type="submit"><%= link_to 'Admin'  , admin_path %></button></li>
+<ul class="nav nav-pills">
+  <li><%= link_to 'Home'     , root_path %></li>
+  <li><%= link_to 'Log In'   , new_user_session_path %></li>
```

Comment:
We should only show log in when they are logged out, and only show log out when they are logged in.